### PR TITLE
Parameter manager events

### PR DESCRIPTION
**Pull request by cmutel**
Last PR for completing events for parameters

### CODE REVIEW

File: bw2data/parameters.py [hunk 1]
```diff
@@ -1361,14 +1367,16 @@ def reformat(ds):
...
+            for pp in ProjectParameter.select().where(ProjectParameter.name << tuple(new)):
+                pp.delete_instance()
+            for dataset in data:
+                ProjectParameter.create(**dataset)
             Group.get_or_create(name="project")[0].expire()
             ProjectParameter.recalculate()
 
     def new_database_parameters(self, data, database, overwrite=True):
-        """Efficiently and correctly enter multiple parameters. Deletes **all** existing database parameters for this database.
+        """Correctly enter multiple parameters. Deletes **all** existing database parameters for this database.
```
Comment (jsvgoncalves) [inline bw2data/parameters.py:1379]: Losing efficiency? :(

**Review by jsvgoncalves — CHANGES_REQUESTED** [review]
Tests are failing now

File: bw2data/parameters.py (hunk 1 above)
Comment (cmutel) [reply bw2data/parameters.py:1379]: Yes, we need to replace a single `delete` SQL statement with a loop as we only capture events on `.save()` and `.delete_instance()`. See [the Peewee docs](https://docs.peewee-orm.com/en/latest/peewee/playhouse.html#signals).

**Comment by cmutel** [comment]
@jsvgoncalves Any idea why the tests aren't running anymore?

**Review by jsvgoncalves — APPROVED** [review]
> Any idea why the tests aren't running anymore?

Standard behaviour on this repo, as far as I'm aware. But I ran the CI manually: https://github.com/brightway-lca/brightway2-data/actions/runs/11949161759

I'll squash and merge this.

**Comment by cmutel** [comment]
Sorry, just so I understand it - does that mean you manually requested the tests to run before? Could I have triggered this as well?

**Comment by jsvgoncalves** [comment]
> Sorry, just so I understand it - does that mean you manually requested the tests to run before? Could I have triggered this as well?

Yes, by clicking `Run Workflow` here: https://github.com/brightway-lca/brightway2-data/actions/workflows/python-test.yml

**Pull request merged**